sanic/request.py

```diff
@@ -114,6 +114,8 @@ def args(self):
     @property
     def cookies(self):
         if self._cookies is None:
+            if 'cookie' in self.headers: #HTTP2 cookie header
```

Comment:
I think two `if` here are redundant, this could be a bit simplified&rewrited
`cookie = self.headers.get('Cookie') or self.headers.get('cookie')
if cookie:
    cookies = SimpleCookie()
    cookies.load(cookie)
`